lib/portlayer/attach/communication/lazy.go

```diff
@@ -50,3 +50,11 @@ func (l *LazySessionInteractor) Initialize() (SessionInteractor, error) {
 	}
 	return l.si, nil
 }
+
+// Cleanup returns either an initialized connection, or nil if it was never initialized
+func (l *LazySessionInteractor) Cleanup() (SessionInteractor, error) {
```

Comment:
Maybe want to name this 'conn' or 'session'?  Doesn't look like it needs to be exported either.